docs/docker.md

```diff
@@ -15,7 +15,7 @@ $ docker run -d \
     -p 5984:5984 \
     -e COUCHDB_USER=admin -e COUCHDB_PASSWORD=password \
     -v $HOME/.cozy-stack-couch:/opt/couchdb/data \
-    couchdb:3.2
+    couchdb:3.3
```

Comment:
No `couchdb:3.3` tag available yet: https://hub.docker.com/_/couchdb/tags

Reply:
The tag is now available